src/Controller/SignupController.php

```diff
@@ -25,31 +26,29 @@ public function __construct(ViewRenderer $viewRenderer)
     }
 
     public function signup(
-        RequestInterface $request,
-        IdentityRepositoryInterface $identityRepository,
         EntityWriter $entityWriter,
+        IdentityRepositoryInterface $identityRepository,
+        LoginForm $loginForm,
```

Comment:
I think `LoginForm` is a data-class and should not be created by container. 
In that case probably better to move translation to the template

Reply:
done.